### PR TITLE
[stable/artifactory] adding support for edge node and replicator

### PR DESCRIPTION
**Pull request by jainishshah17**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: stable/artifactory/README.md [hunk 1]
```diff
@@ -105,6 +126,8 @@ The following table lists the configurable parameters of the artifactory chart a
 | `artifactory.service.type`| Artifactory service type | `ClusterIP` |
 | `artifactory.externalPort` | Artifactory service external port | `8081`   |
 | `artifactory.internalPort` | Artifactory service internal port | `8081`   |
+| `artifactory.internalPortReplicator` | Replicator service internal port | `6061`   |
```
Comment (unguiculus) [inline stable/artifactory/README.md]: Could you fix the alignment of the table?

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: stable/artifactory/README.md (hunk 1 above)
Comment (jainishshah17) [reply stable/artifactory/README.md]: @unguiculus done

**Comment by jainishshah17** [comment]
@unguiculus Can you take a look.

**Comment by jainishshah17** [comment]
@unguiculus This chart is ready.

**Comment by eldada** [comment]
@unguiculus - can you give us a status on this?

**Review by unguiculus — APPROVED** [review]

**Pull request merged**